apps/web/components/eventtype/EventWebhooksTab.tsx

```diff
@@ -20,6 +22,7 @@ export const EventWebhooksTab = ({ eventType }: Pick<EventTypeSetupProps, "event
   const { t } = useLocale();
 
   const utils = trpc.useContext();
+  const formMethods = useFormContext<FormValues>();
```

Comment:
We now need form methods in order to set discretely the different fields in the event type metadata